scripts/managed-services/get-kafka-instance-state/metadata.yaml

```diff
@@ -0,0 +1,48 @@
+file: get-kafka-instance-state.sh
+name: managed-services/get-kafka-instance-state
```

Comment:
This is causing a problem when running with the testing tooling

```
ERRO[0004] error from backplane: 
 Status Code: 500
 Message: Unable to create test job on the cluster, error: invalid canonical name, can't have more than 2 slashes
 ```
 I think it should just be `get-kafka-instance-state`

Reply:
I've changed that now